### PR TITLE
downgrade target-s3 to development status

### PR DESCRIPTION
**Pull request by pnadolny13**
A user in slack was stuck on this target so I was testing it out and noticed that its still in development. I opened some issues to track what I noticed on https://github.com/crowemi/target-s3/issues and pinged the maintainer to see what the development plan is https://meltano.slack.com/archives/C01RKUVUG4S/p1680544334254469?thread_ts=1666981269.271539&cid=C01RKUVUG4S.

### CODE REVIEW

**Review by tayloramurphy — APPROVED** [review]

**Comment by pnadolny13** [comment]
I changed this to a draft PR - from a message in slack from the maintainer:

> Yes, this is the primary target we use in production (parquet/json). However, I’ve been using/modifying the target in our private repository — I’ll can get the latest pushed to GitHub, I think that will solve a couple of the issues you submitted

I tested some of the updates and its solves most of the issues I'm seeing. Before this merges we'll probably want to refresh the metadata.

**Comment by pnadolny13** [comment]
Closing because I created https://github.com/meltano/hub/pull/1267 instead to update it in place.

**Pull request closed without merging**
